src/coreclr/vm/interpexec.cpp

```diff
@@ -11,6 +11,110 @@ typedef void* (*HELPER_FTN_PP)(void*);
 
 thread_local InterpThreadContext *t_pThreadContext = NULL;
 
+FrameDataFragment *frame_data_new_fragment(size_t size)
```

Comment:
We should transform all this fragment/allocator code to use ctors and instance methods on the struct.

This code should follow CoreCLR coding guidelines. Naming like `NewFragment` instead of `new_fragment`. You might want to prefix name for vars of pointer to structs like `pFrag`. Parenthesis always on new line.

Reply:
Moved to `src/coreclr/vm/interpframeallocator.h`/`.cpp` and updated according to the CoreCLR guidelines. I also noticed that both `NULL` and `nullptr` are used in the code. What is the rule for this?

Reply:
We prefer `nullptr` in large chunks of new code.

Otherwise, we prefer consistency with surrounding code.